ntttcp/ntttcp.Config.json

```diff
@@ -0,0 +1,77 @@
+{
+    "NtttcpDefault": {
+        "Iterations"  : 5,
+        "StartPort"   : 50002,
+        "tcp" : {
+            "BufferLen"   : [65536],
+            "Connections" : [1, 64],
+            "Options"     : ""
+        },
+        "udp" : {
+            "BufferLen"  : [1472],
+            "Connections": [1, 64],
+            "Options"    : ""
+        },
+        "Warmup"      : 2,
+        "Cooldown"    : 2,
+        "Runtime"        : 60,
+        "RecvOptions" : "",
+        "SendOptions" : ""
+    },
+
+    "NtttcpAzure": {
+        "Iterations"  : 6,
+        "StartPort"   : 50002,
+        "tcp" : {
+            "BufferLen"   : [65536],
+            "Connections" : [1, 64, 256],
+            "Options"     : ""
+        },
+        "udp" : {
+            "BufferLen"   : [1472],

```

Comment:
we might want to add a testcase for bufferlen 1372 to validate throughtput without getting into the state of fragmentation /hash collisions.. Ok to fast follow